README.md

```diff
@@ -300,6 +300,9 @@ prototype. Takes an optional name which is used when emitting timings.
 Render the component. Returns a proxy node if already mounted on the DOM. Proxy
 nodes make it so DOM diffing algorithms leave the element alone when diffing.
 
+### `component.rerender()`
+Re-run `.render` using the last `arguments` that were passed to the `render` call.  Useful for triggering component renders if internal state has change.  Arguments are automatically cached under `this._arguments` (🖐 hands off, buster! 🖐).
```

Comment:
should be `has changed` i think, or just `changed` .

Reply:
Good catch.